test_ptrack/test_views.py

```diff
@@ -48,21 +54,22 @@ class TestTrackingPixel(ptrack.TrackingPixel):
     def record(self, request, *args, **kwargs):
         test_record(*args, **kwargs)
 
+
 ptrack.tracker.register(TestTrackingPixel)
 
 
 def generate_template_tag_param_str(*args, **kwargs):
     param_str = u""
     for arg in args:
         param_str += u"'{}' ".format(arg)
-    for key, value in kwargs.iteritems():
+    for key, value in kwargs.items():
         param_str += u"{}='{}' ".format(key, value)
     return param_str
 
 
-class PtrackViewsTest(WebTest):
+class PtrackViewsTest(TestCase):
```

Comment:
`django-webtest` does not seem to work with Django 2.x. Switching to the default Django test client appears to work fine.